core/backend/src/IModelTransformer.ts

```diff
@@ -746,46 +749,75 @@ export class IModelTransformer extends IModelExportHandler {
     return targetElementAspectProps;
   }
 
+  /** The directory where schemas will be exported, a random temporary directory */
+  protected _schemaExportDir: string = path.join(KnownLocations.tmpdir, Guid.createValue());
+
+  /** Export a schema that was deserialized to a given path */
+  private shouldExportDeserializedSchema(schemaPath: string): boolean {
+    let schemaSrc: string;

```

Comment:
Doesn't the call to querySchemaVersion on the target eliminate the need for this method?

Reply:
this method is for the filter on schema paths that are exported by `nativeDb.exportSchemas` in `processSchemas`. If we can't rely on a naming convention for the exported files, then we need to read the file for the schema name and version to know what we're importing from the db. We could query the sourceDb for its schema versions but I don't think we can correlate it to the exported files without doing the same amount of work.